packages/nextjs/app/builders/0x74370B567f5c65bef0428B9c78df5C691B632Cf7/components/BuilderProfile.tsx

```diff
@@ -0,0 +1,79 @@
+import Image from "next/image";
+import SkillBadge from "./SkillBadge";
+import SocialLinks from "./SocialLinks";
+import { Award, FileText, Heart } from "lucide-react";
+import { Address } from "~~/components/scaffold-eth/Address/Address";
+
+interface BuilderProfileProps {
+  name: string;
+  avatarUrl: string;
+  bio: string;
+  ethAddress: string;
+  skills: Array<{ name: string; proficiency: number }>;
+  hobbies: string[];
+}
+
+const BuilderProfile = ({ name, avatarUrl, bio, ethAddress, skills, hobbies }: BuilderProfileProps) => {
+  return (
+    <div className="container mx-auto px-4 py-8 max-w-4xl">
+      <div className="card bg-white/30 backdrop-blur-sm border border-purple-100/20 shadow-lg"></div>
+      <div className="absolute inset-0 bg-gradient-to-br from-purple-50/50 to-purple-100/30 -z-10" />
+
+      <div className="flex flex-col items-center space-y-4 pt-8 pb-6">
+        <div className="avatar">
+          <div className="w-32 rounded-full ring ring-[#385184] ring-offset-base-100 ring-offset-2 shadow-lg">
+            <Image src={avatarUrl} alt={name} className="object-cover" />
```

Comment:
There's an error related to this image that's shown both in the browser and in NextJS console. Can you fix it please? Thanks